### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by jamigibbs**
This will add the category dropdown in the navigation and an about page also linked in the nav bar.

### CODE REVIEW

File: client/store/category.js [hunk 1]
```diff
@@ -18,21 +15,32 @@ const initialState = {
...
-
 /**
  * THUNK CREATORS
  */
+
+export const getAllParentCategoriesThunk = () => {
+  return async (dispatch) => {
+    const { data } = await axios.get('/api/categories/all/parent')
+    dispatch(getAllParentCategories(data))
+
```
Comment (chansiky) [inline client/store/category.js]: this function needs 2 ending curly brackets. - 
1 for thunk
1 for the async(dispatch)

File: client/store/category.js [hunk 2]
```diff
@@ -43,10 +51,18 @@ export const getPopularCategoriesThunk = () => {
 /**
  * REDUCER
  */
+
+const initialState = {
```
Comment (chansiky) [inline client/store/category.js]: this is a duplicate 'initialState' see line X.
I would move

File: client/store/single-category.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ export default function(state = initialState, action) {
     case FIND_ALL_ITEMS_IN_A_CATEGORY:
         const paths = action.content.filter(item => item.type === 'Path')
         const resources = action.content.filter(item => item.type === 'Resource')
+        console.log('name', action.name)
```
Comment (chansiky) [inline client/store/single-category.js]: console.log - remove if its not necessary

**Comment by jamigibbs** [comment]
All those issues should be fixed. :)

**Pull request merged**